bom/pom.xml

```diff
@@ -46,11 +45,6 @@ THE SOFTWARE.
 
   <dependencyManagement>
     <dependencies>
-      <dependency>
-        <groupId>com.github.spotbugs</groupId>
-        <artifactId>spotbugs-annotations</artifactId>
```

Comment:
why remove this?

Reply:
It is part of the parent pom- So I thought removing it would not have an effect.

Reply:
This actually removes `spotbugs` from the published bom as we are using [`bom` mode for the maven flatten plugin](https://www.mojohaus.org/flatten-maven-plugin/flatten-mojo.html#flattenMode)

(Idouble checked by running `mvn install` and looked at the resultant file in my local repo)

Reply:
Ok, thanks for double checking. Can I remove the version, so that we at least have the same as the parent pom?

Reply:
And I should then introduce the optional tag, because we do not need it in runtime. Would then also be the recommanded dependency, as stated here: https://spotbugs.readthedocs.io/en/stable/migration.html#com-google-code-findbugs-findbugs-annotations

Reply:
>  Can I remove the version

that I am not sure about you would have to test it, but I have a feeling the answer is no.

> And I should then introduce the optional tag

no, should be handled where it is used -> https://github.com/jenkinsci/plugin-pom/blob/e2c98043e8f86fd8c71e134030a3e3829ee20d75/pom.xml#L205-L210

Reply:
You were right. I have to specify the version.
So maybe introducing the property to the parent pom would be the right approach? I will create a PR for that.